packages/webpack/src/excludeLoader.js

```diff
@@ -1 +1 @@
-module.exports = (source) => source
+module.exports = (/** @type {any} */ source) => source
```

Comment:
It's any, because only the first loader gets a string, all other loaders may use whatever object another loader in a sequence can consume and the final one must return a string or Source type.

wrote the description.